La llamada remota se ejecuta normalmente pero envuelta en un Thread que puede pasarse 
aunque no se haya regresado de la llamada. Ello provocará excepciones en el cliente
